precompiles/utils/src/input_parsers.rs

```diff
@@ -24,7 +24,7 @@ pub fn parse_account(input: &[u8]) -> Result<H160, ExitError> {
 	const TOTAL_SIZE_BYTES: usize = PADDING_SIZE_BYTES + ACCOUNT_SIZE_BYTES;
 
 	if input.len() != TOTAL_SIZE_BYTES {
-		log::trace!(target: "staking-precompile",
+		log::trace!(target: "precompile-utils",
```

Comment:
how do you use this? Do you have to pass in some flag = target when running things?

Reply:
yes

Reply:
So there's a different flag for this than `staking-precompile` or `crowdloan-precompile`? Does that mean this log trace won't show up if I pass either of those flags? If so, could we make this target = `staking-precompile` OR `crowdloan-precompile`?

Reply:
Yeah I can do that definitely!

Reply:
Actually I dont think that trace log makes sense in a common funciton. I can either move the trace log to each of the modules in the case of an error if that's better